clang/include/clang/Basic/BuiltinsNVPTX.def

```diff
@@ -218,6 +218,10 @@ BUILTIN(__nvvm_sin_approx_f, "ff", "")
 BUILTIN(__nvvm_cos_approx_ftz_f, "ff", "")
 BUILTIN(__nvvm_cos_approx_f, "ff", "")
 
+// Tanh
+
+TARGET_BUILTIN(__nvvm_tanh_approx_f, "ff", "", AND(SM_75,PTX70))
```

Comment:
Should it be `SM_80` to align with the driver option description and libclc implementation?

Reply:
Yes, probably it's better.